llama_stack/cli/model/list.py

```diff
@@ -5,12 +5,52 @@
 # the root directory of this source tree.
 
 import argparse
+import os
+import time
 
 from llama_stack.cli.subcommand import Subcommand
 from llama_stack.cli.table import print_table
+from llama_stack.distribution.utils.config_dirs import DEFAULT_CHECKPOINT_DIR
 from llama_stack.models.llama.sku_list import all_registered_models
 
 
+def _convert_size_to_gb(size):
+    return size / (1024**3)
+
+
+def _get_model_size(model_dir):
+    model_size = 0
+    for dirpath, _, files in os.walk(model_dir):
+        for file in files:
+            file_path = os.path.join(dirpath, file)
+            model_size += os.path.getsize(file_path)
+    return model_size
```

Comment:
nit: I think you could use pathlib to simplify this block
```suggestion
    return sum(f.stat().st_size for f in Path(model_dir).rglob('*') if f.is_file())
```